integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -107,6 +107,7 @@ window.addEventListener("load", () => {
             link_default_target: "",
             document_base_url: tinymceConfig.getAttribute("data-webapp-url"),
             relative_urls: false,
```

Comment:
I guess we could remove this line then?

Reply:
Which one?

Reply:
The `relative_urls: false`
It does not really apply anymore, we keep relative urls now, so I guess it could be a bit confusing if we leave it in